### PR TITLE
feat: bump `deno.json` & `deno.jsonc` by default

### PR DESCRIPTION
**Pull request by ryoppippi**
### Description

Just like `jsr.json` and `jsr.jsonc`, bump `deno.json` & `deno.jsonc` by default!

### Linked Issues
#27 #34

### CODE REVIEW

File: src/update-files.ts [hunk 1]
```diff
@@ -44,6 +44,7 @@ async function updateFile(relPath: string, operation: Operation): Promise<boolea
     case 'bower.json':
     case 'component.json':
     case 'jsr.json':
+    case 'deno.json':
```
Comment (antfu) [inline src/update-files.ts:47]: Is here missing `deno.jsonc`?

Comment (ryoppippi) [reply src/update-files.ts:47]: `jsr.jsonc` isn't here.
I think `jsonc` and `json` are differnt

Comment (antfu) [reply src/update-files.ts:47]: I am not sure if it was correct. But let's have it for now, as jsonc would be a bit tricky to deal with correctly and I'd have the bandwidth to look into it right now.

Comment (ryoppippi) [reply src/update-files.ts:47]: When I found a bug for that, I'll contribute it again!

**Comment by ryoppippi** [comment]
Maybe `--ignore-scripts` might help you

**Comment by wobsoriano** [comment]
@ryoppippi awesome, that worked

**Pull request merged**